layouts/partials/encounter/single_header.html

```diff
@@ -0,0 +1,31 @@
+<div class="h-112 lg:h-96 xl:h-80 bg-no-repeat bg-cover" style='background-image: url({{ .Params.banner_image }});'>
+  <div class="responsive-container z-10 relative flex items-center h-full mt-4 lg:mt-0">
+    <div class="grid grid-cols-12 lg:gap-8 w-full items-center mt-8 lg:mt-0">
+      <div class="col-span-12 lg:col-span-9 text-white responsive-container">
+        <div class="flex items-center mb-7">
+          <a href="/encounters" class="flex flex-row items-center" >
+            <div class="card-title filter drop-shadow-lg-encounter">{{ .Params.series_name }} &raquo; {{ .Params.tier_name }}</div> 
+          </a>
+        </div>
+        <div>
+          <div class="flex items-center mb-4">
+            <img class="inline mr-8 filter drop-shadow-lg-encounter" src="/theme-assets/encounters.svg" />
+            <div class="font-bold filter drop-shadow-lg-encounter text-2xl lg:text-6xl uppercase">{{ .Params.fight_title }}</div>
```

Comment:
Ditto with changing this back to `{{ .Title }}`

Reply:
Resolved!